qemu-cc/qemu-cc.spec

```diff
@@ -0,0 +1,198 @@
+Name: qemu-cc
+Version: v2.9.0+git.a2e6957b6f
+Release: 0
+Source0: %{name}-%{version}.tar.gz
+Source1: qemu-cc-rpmlintrc
+BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}
+
+Summary  : OpenBIOS development utilities
+Group    : Development/Tools
+License  : BSD-2-Clause BSD-3-Clause GPL-2.0 GPL-2.0+ LGPL-2.0+ LGPL-2.1
+
+Requires: qemu-cc-bin
+Requires: qemu-cc-data
+Requires: qemu-cc-doc
+BuildRequires : automake
+BuildRequires : bison
+BuildRequires : flex
+BuildRequires : glib2-devel
+BuildRequires : libattr-devel
+BuildRequires : libcap-devel
+BuildRequires : libcap-ng-devel
+BuildRequires : libtool
+BuildRequires : libtool-ltdl-devel
+BuildRequires : libtool
+BuildRequires : m4
+BuildRequires : numactl-devel
+BuildRequires : python-devel
+BuildRequires : zlib
+BuildRequires : pkgconfig(pixman-1)
+Patch1: configure.patch
+
+%undefine _missing_build_ids_terminate_build
+%define  debug_package %{nil}
+
+%description
+===========
+QEMU is a generic and open source machine & userspace emulator and
+virtualizer.
+
+%package bin
+Summary: bin components for the qemu-cc package.
+Group: Binaries
+Requires: qemu-cc-data
+
+%description bin
+bin components for the qemu-cc package.
+
+
+%package data
+Summary: data components for the qemu-cc package.
+Group: Data
+
+%description data
+data components for the qemu-cc package.
+
+%package doc
+Summary: docs for the qemu-cc package.
+Group: Data
+
+%description doc
+Docs for the qemu-cc package.
+
+%prep
+ls -la /home/abuild/rpmbuild/SOURCES/
+
+%setup -q
+%patch1 -p1
+
+%build
+export LANG=C
+%configure --prefix=/usr --disable-tools --disable-libssh2 --disable-tcmalloc --disable-glusterfs \
```

Comment:
This looks neat, but makes maintenance more difficult: could you split this so each option is put on a separate line:

```
--option-1 \
--option-2 \
  :
```